src/Microsoft.Health.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -15,11 +16,13 @@
 using Microsoft.Health.SqlServer.Features.Routing;
 using Microsoft.Health.SqlServer.Features.Schema;
 using Microsoft.Health.SqlServer.Features.Schema.Extensions;
+using Microsoft.Health.SqlServer.Features.Schema.Messages.Get;
 
 namespace Microsoft.Health.SqlServer.Api.Controllers
 {
     [HttpExceptionFilter]
     [Route(KnownRoutes.SchemaRoot)]
+    [SuppressMessage("Reliability", "CA2007:Consider calling ConfigureAwait on the awaited task", Justification = "ASP.NET Core does not have custom synchronization context.")]
```

Comment:
I think it's probably reasonable to add `ConfigureAwait(false)` here too, as this is specifically for ASP.NET Core which does not include a custom synchronization context. Furthermore, it reinforces that the code here is agnostic to any specific scheduling needs